### ISSUE TITLE
Increased apolipoprotein levels

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5988]
**Preferred term label:** Increased apolipoprotein levels

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of apolipoprotein in the blood is above the normal range., PMID: 6099394

**Parent term (use hpo.jax.org/app)** Abnormal apolipoprotein level HP:0025201

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
This could be a parent term for: Elevated apolipoprotein B level HP:0031798; Elevated apolipoprotein A-II level HP:0031800; Elevated apolipoprotein A-IV level HP:0025202

**Comment by pnrobinson** [comment]
I think we would just put these subterms under this: https://hpo.jax.org/app/browse/term/HP:0025201
